sample/Ookii.Dialogs.WinForms.Sample/Ookii.Dialogs.WinForms.Sample.csproj

```diff
@@ -1,41 +1,27 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D52F3B1C-03B3-41B1-A15E-E23FF6877939}</ProjectGuid>
     <OutputType>WinExe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Ookii.Dialogs.WinForms.Sample</RootNamespace>
-    <AssemblyName>Ookii.Dialogs.WinForms.Sample</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <!--<TargetFramework>net45</TargetFramework>-->
+    <!--<TargetFramework>netcoreapp3.1</TargetFramework>-->
+    <TargetFramework>net5.0-windows</TargetFramework>
```

Comment:
Could we multi-target to all three TFMs? (Would it work?) or do we need separate example projects for each TFM?

Reply:
Multi-targeting front-end is not possible, however the same code files can be used via the Add as link:
![image](https://user-images.githubusercontent.com/40712699/107649018-ad5dfc00-6c85-11eb-974f-3ad1b7bc9d6f.png)

 This would then generate a single project file with targetting a specific TFM:
```xml
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net5.0-windows</TargetFramework>
    <UseWindowsForms>true</UseWindowsForms>
  </PropertyGroup>

  <ItemGroup>
    <None Remove="ookii.ico" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\..\src\Ookii.Dialogs.WinForms\Ookii.Dialogs.WinForms.csproj" />
  </ItemGroup>

</Project>
```

All I need are the names for these projects if you wish that I include them in the PR?

Reply:
Thanks for going the extra mile!

I'd suggest a similar [naming convention I used in the samples for Prism.Logging.Serilog](https://github.com/augustoproiete/prism-logging-serilog/tree/master/sample): `sample/tfm/ProjectNameDemoTfm.csproj`

i.e.

- `sample/net45/OokiiDialogsWpfDemoNet45.csproj`
- `sample/netcoreapp3.1/OokiiDialogsWpfDemoNetCoreApp31.csproj` 
-  `sample/net5.0/OokiiDialogsWpfDemoNet50.csproj`

Reply:
Added the suggested projects targeting different TFMs.